app/src/main/res/xml/preferences.xml

```diff
@@ -158,6 +159,17 @@
             search:summary=""
             android:title="@string/settings_alarm_sleep_timer" />
 
+        <SeekBarPreference
+            android:id="@+id/gradually_increase_volume_seconds"
+            android:defaultValue="0"
+            app:min="0"
+            android:max="12"
```

Comment:
Add `app:updatesContinuously="true"` for it to update when user drags the slider.